pandas/io/parsers.py

```diff
@@ -1423,6 +1424,26 @@ def __init__(self, kwds):
         # keep references to file handles opened by the parser itself
         self.handles = []
 
+    def _confirm_parse_dates_presence(self, columns):
+        """
+        if user has provided names for parse_dates, check if those columns
+        are available.
+        """
+        if isinstance(self.parse_dates, list):
+            cols_needed = self.parse_dates
+        elif isinstance(self.parse_dates, dict):
+            cols_needed = chain(*self.parse_dates.values())
+        else:
+            cols_needed = []
```

Comment:
so this is a single scalar column, why is it not needing validation?

Reply:
> parser_dates: bool or list of int or names or list of lists or dict, default False
according to the read_csv doc, this can't be a single scalar.